source/types/trinary.cpp

```diff
@@ -278,9 +279,19 @@ tritsToTrytes(const Trits& trits, std::size_t length) {
 
 Types::Trits
 intToTrits(const int64_t& value) {
-  Types::Trits trits;
-  uint64_t     absoluteValue = std::abs(value);
+  if (value == 0)
+    return {};
 
+  Types::Trits trits;
+  int          sign          = (value > 0) - (value < 0);
+  uint64_t     absoluteValue = value * sign;
```

Comment:
does that optimize compared to a std::abs? I'd prefer keep std::abs for code clarity, except if it brings actual improvement.

Reply:
ok, nevermind, sign is reused in the loop